extensions/wrapper/clients/typescript-client/src/oauth2/RequestUtils.ts

```diff
@@ -0,0 +1,59 @@
+import {createUrlEncodedParamsString} from '../utils/HttpUtils';
+
+export function buildTokenRequestFormData(
+    clientId: string,
+    clientSecret: string,
+): string {
+    const formData = {
+        grant_type: 'client_credentials',
+        client_id: clientId,
+        client_secret: clientSecret,
+    };
+
+    return createUrlEncodedParamsString(formData);
```

Comment:
I mean, if you have this in the fetch call:

````ts
body: createUrlEncodedParamsString({
    grant_type: 'client_credentials',
    client_id: clientId,
    client_secret: clientSecret,
})
```

isn't that super transparent?